mne/tests/test_epochs.py

```diff
@@ -145,6 +145,19 @@ def test_epoch_combine_ids():
         # should probably add test + functionality for non-replacement XXX
 
 
+def test_epoch_hierarchical_ids():
+    """Test multi-index/hierarchical indexing
+    """
+    raw, events, picks = _get_data()
+    for preload in [False]:
```

Comment:
yes, it had a historical context, i.e., there were more items in that list during intial development for debugging.